editor/editor_settings.cpp

```diff
@@ -764,21 +764,15 @@ void EditorSettings::create() {
 		}
 
 		if (dir->change_dir("templates") != OK) {
-			dir->make_dir("templates");
+			dir->make_dir_recursive("templates");
```

Comment:
Unless I'm missing something, `make_dir_recursive()` doesn't seem needed to me here, we've just created the `data_dir` above, and anyway `"templates"` doesn't include the relevant information to recursively make the absolute path to the target folder.

Reply:
That said it doesn't *hurt* either, I suppose it does exactly the same logic when there's no recursiveness needed.

Reply:
you're right, I'll change it